taos-ws-py/examples/pandas-read-sql.py

```diff
@@ -0,0 +1,7 @@
+import pandas
+from sqlalchemy import create_engine
+import taosws
+
+engine = taosws.connect("taos://localhost:6041")
```

Comment:
current pattern is native use taos://...:6030, REST use taosrest://...:6041, should we use taosws://...:6041 to make user explicitly know using websocket?

Reply:
I guess here we should demo how create_engine work with py connector?

Reply:
> current pattern is native use taos://...:6030, REST use taosrest://...:6041, should we use taosws://...:6041 to make user explicitly know using websocket?

Aggreed.



> I guess here we should demo how create_engine work with py connector?

Codes ready. Need to publish new taospy version.